### ISSUE TITLE
1.8.1

### ISSUE THREAD

**Issue by Vtato** [moneymanagerex/moneymanagerex#6993]
MMEX version:
 - [ ] 1.7.0
 - [X] 1.8.0
 - [ ] 1.8.1 No puedo abrir archivo cifrado al cambiar a esta versión
 - [ ] 1.8.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Please describe your issue with details.
Add screenshot or other files if needed.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

Expected result:

Actual result:

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by n-stein** [comment]
Duplicate of #6971.
